docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+# Z-JSON Instrument Specification
+
+This documentation describes and explains the Z-JSON Instrument. It is supported by a template and an example,
+both of which can be found in the same directory.
+
+The Z-JSON Instrument is the JSON representation of a LORIS instrument. Describing an instrument as a JSON
+object will allow the instrument to be language-independent, human readable, and shareable across different
+platforms. This schema will become an important part of the LORIS API and the New Instrument Builder, and was built
+to be compatible with both the [ReproNim JSON-LD Form Schema](https://github.com/ReproNim/schema-standardization)
```

Comment:
interoperable?

Reply:
..but what does interoperable mean in this context? How do they interoperate?

Reply:
it means that we built this schema with the intention to easily mappable to both JSON-LD and NDAR via converters that will also be created to be included in the package. how are you looking for me to word this on the spec? @driusan

Reply:
Hm.. I'm not sure about wording, but "compatible" doesn't really say much in this context.

How about the wording that you just said instead of "compatible"? "...was built with the intention to be easily mappable to both ReproNim's JSON-LD schema and NDAR's schema via converters"?